devel/codemod/Portfile

```diff
@@ -25,6 +25,6 @@ checksums           rmd160  39b8461ab9e16c68a8605fb18a1e8b6c46242bd2 \
                     sha256  dc72368bc95fab74373f700a32682d12d308828215c973674c06d63da3494dbb \
                     size    17756
 
-python.versions     311
+python.versions     313
 
 depends_build-append    port:py${python.version}-setuptools
```

Comment:
the default for Python 3.7 is to use the `setuptools` PEP517 backend, so you don't need to explicitly list the dependency. Could you remove it?